guide/english/book-recommendations/python/index.md

```diff
@@ -67,5 +67,8 @@ You'll learn many of the first princliples for data science tools and how algori
 - [Python in Education by Nicholas H. Tollervey[OREILLY] [FREE]](https://www.oreilly.com/programming/free/files/python-in-education.pdf)
 - [How to think like a Computer Scientist by Allen Downey[V 2.0.17] [FREE]](http://greenteapress.com/thinkpython/thinkpython.pdf)
 - [A Byte of Python[FREE]](https://python.swaroopch.com)
+- [QuantEcon - Python Lectures [FREE]] (https://lectures.quantecon.org/py/index.html)
```

Comment:
Needs formatting changes as mentioned in https://github.com/freeCodeCamp/freeCodeCamp/pull/23752#pullrequestreview-178584142